### ISSUE TITLE
modularize rv_multipanel_plot

### ISSUE THREAD

**Issue by petigura** [California-Planet-Search/radvel#125]
rv_multipanel_plot is quite long and hard to tweak, modularize.

- Should be able to make a phasefolded plot for a single planet
- Should be able to make a timeseries plot by itself. 

API for  rv_multipanel_plot 

```
def rv_multipanel_plot():
    fig, axL = subplots() # option to pass in custom-sized figures
    sca(ax_rv)
    plot_rv()

    sca(ax_phase[0])    
    plot_phase()

    etc

```

**Comment by bjfultn** [comment]
Adding bug reported by @sealauren into this issue.

Minor bug: ysigma_scale no longer works as it should; it only adjusts the upper y-limit, not the lower y-limit.

**Comment by sblunt** [comment]
Addressed in Version 1.1.10
